components/support/Card.tsx

```diff
@@ -0,0 +1,58 @@
+import styles from '../../styles/SupportCard.module.css'
+import org from '../../public/assets/temp/org.svg'
+import star_filled from '../../public/assets/icon/star-filled.svg'
+import star_outlined from '../../public/assets/icon/star-outlined.svg'
+import Image from 'next/image'
+import { useState } from 'react'
+
+interface Props {
+  data: { [key: string]: string }
+  clickHandler?: (event: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+export default function Card(props: Props) {
+  const [liked, setLiked] = useState(true) // TODO: think of where this data should come from
+  const [displayStar, setDisplayStar] = useState(true) // retrieved from auth state
```

Comment:
I thought the idea was going to be that if they liked it, then they would be redirected to login/register.